README.md

```diff
@@ -124,9 +124,9 @@ Release notes will be automatically generated by the CI setup when releasing a p
 
 ### Manual Deployments
 
-Based on the git branch and/or tags, builds get deployed automatically by [GitHub Actions](https://github.com/wireapp/wire-webapp/actions). In case the actions are not working, a manual deployment can be triggered using `yarn deploy`.
+Based on the git branch and/or tags, builds get deployed automatically by [GitHub Actions](https://github.com/wireapp/wire-webapp/actions). In case the actions are not working, a manual deployment can be triggered using `yarn deploy`. It will take a current branch and will deploy it to the environemnt which is configured [here](./.elasticbeanstalk/config.yml). To deploy any branch to specific environment simply run `yarn deploy wire-webapp-envname` (e.g. `yarn deploy wire-webapp-qa`).
```

Comment:
```suggestion
Based on the git branch and/or tags, builds get deployed automatically by [GitHub Actions](https://github.com/wireapp/wire-webapp/actions). In case the actions are not working, a manual deployment can be triggered using `yarn deploy`. It will take the current branch and will deploy it to the environment which is configured [here](./.elasticbeanstalk/config.yml). To deploy any branch to a specific environment switch to the branch you want to deploy `git checkout my-branch` and simply run `yarn deploy wire-webapp-envname` (e.g. `yarn deploy wire-webapp-qa`).
```